src/main/stackpack/provisioning/templates/splunk-topology-component-template.json.handlebars

```diff
@@ -14,6 +14,10 @@
         {
             "_type": "Label",
             "name": "stackpack:splunk"
+        },
+        {
+             "_type": "Label",
+             "name": "splunk-instance:{{instanceName}}",
```

Comment:
This handlebars file is imported in the shared template. This will not work for the second/next installed StackPack instance. This template should be moved to either the instance template or a different approach has to be found.

Reply:
Duplicating an almost generic/shared template seems a bit much, is there any other approach we can take?
I see this more often in StackPacks, that's why I initially only created the one Splunk only view

Reply:
moved `mapping function` to instance-based so we get the exact instance with each instance view.